test/e2e/kubectl/kubectl.go

```diff
@@ -1063,7 +1063,7 @@ metadata:
 		/*
 			Release : v1.9
 			Testname: Kubectl, label update
-			Description: When a Pod is running, update a Label using ‘kubectl label’ command. The label MUST be created in the Pod. A ‘kubectl get pod’ with -l option on the container MUST verify that the label can be read back. Use ‘kubectl label label-’ to remove the label. Kubetctl get pod’ with -l option SHOULD no list the deleted label as the label is removed.
+			Description: When a Pod is running, update a Label using ‘kubectl label’ command. The label MUST be created in the Pod. A ‘kubectl get pod’ with -l option on the container MUST verify that the label can be read back. Use ‘kubectl label label-’ to remove the label. ‘kubectl get pod’ with -l option SHOULD no list the deleted label as the label is removed.
```

Comment:
"SHOULD no list" also seems strange.
How about fixing it also in this PR?

Reply:
Thank you for your reply.
I changed "SHOULD no" to "SHOULD not". Could you review it?